README.md

```diff
@@ -19,6 +19,14 @@
 [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-conan/depshield.svg)](https://depshield.github.io)
 
+> **Huzzah!** Conan is now part of Nexus Repository Manager. Version 3.20 includes the Conan plugin by default. 
+>The plugin source code is now in [nexus-public](https://github.com/sonatype/nexus-public) in [nexus-repository-conan](https://github.com/sonatype/nexus-public/tree/master/plugins/nexus-repository-conan).
+
+> **Filing issues:** Upgrade to the latest version of Nexus Repository Manager 3, to get the latest fixes and improvements, before filing any issues or feature requests at https://issues.sonatype.org/.
+
+> **Upgrading:** If you are using a version prior to 3.20 and upgrade to a newer version you will not be able to install the community plugin. 
+>No other changes are required, and your existing data will remain intact.
```

Comment:
```suggestion
>No other changes are required and your existing data will remain intact.
```
Rationale: This is (trivially) not an Oxford comma, because there are just 2 items, it's just a redundant conjunction.